doc/configuration.rst

```diff
@@ -1861,3 +1863,28 @@ This index file will contain descriptions for the whole gallery as well as for
 each subsection, and a specific toctree for each subsection.
 In particular, sidebars generated using these toctrees might not reflect the
 actual section / folder structure.
+
+.. _missing_doc_ignore:
+
+Ignoring API entries
+====================
+
+By default, ``missing_doc_ignore='.*__.*__'`` ignores files that match this
+regular expression in documenting the usage of API entries within the
+example gallery. This regular expression can be modified to ignore
+any kind of file that should not be considered. The default regular
+expression ignores functions like ``__len__()`` for which it may not be
+desirable to document if they are used in examples.
```

Comment:
There are two reports that are generated, one of the API usage (which API entries are used in which examples) and one of which API entries are not used at all. So if `show_api_usage` is `False`, it only skips the second part. Map of API entries that are used can get quite large because it also connects in examples so we decided to turn it off for MNE. There isn't an option to turn off the unused API entries though. I'll add a comment to clarify.